2022/24xxx/CVE-2022-24387.json

```diff
@@ -1,18 +1,100 @@
 {
```

Comment:
{} is not allowed for {'vendor': {'vendor_data': [{'product': {'product_data': [{'product_name': 'SmarterTrack', 'version': {'version_data': [{'version_affected': '<=', 'version_name': '100.0.8019.x', 'version_value': 'Build 8075'}]}}]}, 'vendor_name': 'SmarterTools'}]}}